cmd/buildah/images.go

```diff
@@ -51,11 +61,26 @@ func imagesCmd(c *cli.Context) error {
 	if c.IsSet("notruncate") {
 		truncate = !c.Bool("notruncate")
 	}
+	json_out := false
+	if c.IsSet("json") {
```

Comment:
you can remove this `if` - when you do `c.Bool`, if the flag isn't set it will still return `false` so no need for this.